### PR TITLE
Add certification tests for the local storage binding and fix module name & package version conflicts

### PR DESCRIPTION
**Pull request by helbing**
# Description

1. add certification tests for the local storage binding
2. fix module name 
3. fix package version conflicts, package antlr4 has two versions

## Issue reference

Please reference the issue this PR will close: #1983

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/certification/bindings/alicloud/nacos/go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
-module servicebusqueue_test
+module github.com/dapr/components-contrib/tests/certification/bindings/alicloud/nacos
 
-go 1.17
+go 1.18
```
Comment (berndverst) [inline tests/certification/bindings/alicloud/nacos/go.mod:3]:
```suggestion
go 1.19
```

File: tests/certification/bindings/localstorage/go.mod [hunk 2]
```diff
@@ -0,0 +1,122 @@
+module github.com/dapr/components-contrib/tests/certification/bindings/localstorage
+
+go 1.18
```
Comment (berndverst) [inline tests/certification/bindings/localstorage/go.mod:3]:
```suggestion
go 1.19
```

File: tests/certification/bindings/alicloud/dubbo/go.mod [hunk 3]
```diff
@@ -1,6 +1,6 @@
-module dubbo_test
+module github.com/dapr/components-contrib/tests/certification/bindings/alicloud/dubbo
 
-go 1.17
+go 1.18
```
Comment (berndverst) [inline tests/certification/bindings/alicloud/dubbo/go.mod:3]:
```suggestion
go 1.19
```

**Review by berndverst — DISMISSED** [review]

**Review by berndverst — APPROVED** [review]

**Comment by addjuarez** [comment]
@helbing can you comment /assign on the issue

**Comment by helbing** [comment]
> @helbing can you comment /assign on the issue

Done

**Pull request merged**